content/en/tracing/connect_logs_and_traces/_index.md

```diff
@@ -121,27 +121,62 @@ This enables automatic trace ID injection for `bunyan`, `paperplane`, `pino`, an
 {{% /tab %}}
 {{% tab ".NET" %}}
 
-The .NET Tracer uses the [LibLog][1] library to automatically inject trace IDs into your application logs. It contains transparent built-in support for injecting into [NLog][2], [Log4Net][3], and [Serilog][4].
+Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.
 
-Enable injection in the .NET Tracer’s [configuration][5] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.
-
-Additionally, injection can be enabled in the code:
+The .NET Tracer uses the [LibLog][2] library to automatically inject trace IDs into your application logs if you are using [Serilog][3], [NLog][4] (version 2.0.0.2000+), or [Log4Net][5]. Automatic injection will only display in your logs after configuring your logger to emit MDC attributes (see examples below). **Note**: Automatic injection only works for logs formatted as JSON.
 
```

Comment:
noob question: What is `MDC`?

Reply:
It stands for MappedDiagnosticsContext, which is a key-value map in the logger that the application can update throughout the program execution. We update the key `dd.trace_id` and `dd.span_id` so their JSON logs can emit the appropriate value at the time of the logging statement.

Reply:
Thanks. Maybe spell it out or reference it to a (new) entry in the APM Glossary.

Reply:
I expanded out to use the full word on its first usage. I think that should be sufficient. But since this is a logging concept and not a tracing one I'm hesitant to add it to the APM Glossary.

Reply:
Sounds good.